0016/README.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 0016
+title: Propose Default workflow
+status: ideation
+authors: Dan <daniel.finneran@gmail.com>
+---
+
+## Summary
+
+This additional feature would allow an end user to specify a workflow to be used against a machine that has no `hardware` profile specified within Tinkerbell. The main feature that this would enable would be for DCIM or hardware discovery where any new unknown machines are automatically prossed by a specified workflow that could perform actions such as:
+
+ - Secure wipe
+ - Forensic analysis
+ - DataCentre inventory
```

Comment:
What kind of DC inventory are you thinking? Presumably an untrusted machine shouldn't have network access to all parts of a DC.

Reply:
The Datacentre inventory would just be whatever tinker bell is capturing from the machines that boot the first time when we don't know about them.

- NICs
- CPU
- Memory 
- GPU
- Disk config
- Is anything installed on the servers
- Firmwares?

Reply:
ah well that doesn't sound like DC inventory, more like node inventory right? Just a wording that might be confusing.